resources/views/components/pulse.blade.php

```diff
@@ -50,7 +47,7 @@
             </main>
         </div>
 
-        @livewireScripts
-        @stack('scripts')
+        {!! Laravel\Pulse\Facades\Pulse::js() !!}
+        @livewireScriptConfig
```

Comment:
> When Livewire detects the @livewireScriptConfig directive, it will refrain from injecting the Livewire and Alpine scripts. If you are using the @livewireScripts directive to manually load Livewire, be sure to remove it. Make sure to add the @livewireStyles directive if it is not already present.